backend/src/graphql/resolver/CommunityResolver.test.ts

```diff
@@ -219,7 +247,9 @@ describe('CommunityResolver', () => {
               {
                 id: 4,
                 foreign: foreignCom1.foreign,
-                publicKey: expect.stringMatching('publicKey-ForeignCommunity'),
+                publicKey: expect.stringMatching(
+                  '08520bf2990274f829d2a2d45c802e4e854a768ed1c757ea99571a24bbfd87b2',
+                ),
```

Comment:
Why not using the variable `foreignCom1.publicKey` instead of hardcoded string?

Reply:
FederatedCommunity.publicKey should be a Buffer type, graphql returns a hex string.
But you right I could make it way better. Save for future test, if the key will be actually validated.